### DISCUSSION TITLE
How to get WP version pipeline was run on?

### DISCUSSION THREAD

**Discussion by zc-devs** [woodpecker-ci/woodpecker#4202] [Request a Feature] [answered]
Is there a way to get a version of Woodpecker an old pipeline was run on?
If no, then convert to feature request, please. Something like
![Sys info](https://github.com/user-attachments/assets/8d9940cc-1c90-4742-bdab-fc10d5f8de0a)

**Comment by qwerty287** [comment, marked as answer]
No. I convert it to a feature

**Comment by zc-devs** [comment]
#4204
